src/components/NavigationItem/index.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   isActive: boolean;
   slug: string;
   title: string;
+  baseUrl?: string;
```

Comment:
Is this really optional?

Reply:
Yes. It could be optional. If nothing is provided, it can be `/` else the provided `baseUrl`